articles.tex

```diff
@@ -303,6 +303,8 @@
 	\item To act as a Judicial Board as defined in \ref{Judicial}.
 	\item To review major projects, as defined in \ref{Expectations of House Members},  presented to them by the Evaluations director.
 	\item To make the final vote regarding conditionals and appeals as defined in \ref{Membership Evaluation}.
+	\item To proactively maintain a safe and welcoming enviornment to House members.
+	\item To respect the privacy of House members who confide in confidence barring circumstances that allude endangerment of ones self or others.
```

Comment:
I'm not too sure it's considered implied. I can absolutely see a future discussion where this passage comes up. I'd prefer explicitly stating this.